vortex-array/src/arrays/constant/mod.rs

```diff
@@ -58,11 +58,10 @@ impl ConstantArray {
         S: Into<Scalar>,
     {
         let scalar = scalar.into();
-        let stats = StatsSet::constant(scalar.clone(), len);
```

Comment:
I think we need the compute functions otherwise constant arrays have no stats

Reply:
As in, we need to implement the compute functions?

Reply:
Yes, implement the compute functions

Reply:
Done. Most of the compute functions already had special case for `is::<ConstantVTable>`